### ISSUE TITLE
Invalid YAML (according to pyyaml)

### ISSUE THREAD

**Issue by dmvaldman** [tangrams/tron-style#144]
In lines [129-131](https://github.com/tangrams/tron-style/blob/gh-pages/layers/ux-sdk-overlay.yaml#L129-L131) of `layers/ux-sdk-overlay.yaml` there is a use of YAML that is technically valid, but pyyaml doesn't like it. When you try to bundle the scene file using the [Tangram bundler](https://github.com/tangrams/bundler), it will error.

**Comment by nvkelso** [comment]
Looks like this affects other styles as well. The solution we used before was:

```
                transition: { show: { time: 0s }, hide: { time: 0s } }
```

from

```
                transition:
                    [show, hide]:
                        time: 0s
```

**Comment by matteblair** [comment]
:+1: This syntax is currently allowed for label transitions in tangram-es, but after discovering the problems it can cause with incomplete YAML parsers we decided to phase it out in favor of the syntax Nathaniel suggested.

**Comment by nvkelso** [comment]
Fixed in 428d546a319c2990a574255b088461057f32627e, and in all other house styles.
